docs/6.coordination/2.procedures.md

```diff
@@ -0,0 +1,249 @@
+Help Desk is the support that FIWARE Lab experts, Nodes Administrators
+and GEs Owners, give to external and internal users. The Help Desk
+activities are structured in 2 main Level of support in order to
+guarantee that agreed SLAs are achieved.
+
+The first level of support is comprised of a team which is in charge of
+managing all FIWARE Lab incoming tickets. This team is also responsible
+for categorizing all incoming tickets in order to guarantee that Backlog
+statistics (<http://backlog.fiware.org/helpdesk/main>) are always
+properly updated. If is not possible for a member of the first level
+support to solve a problem (e.g. a specific Node failure), he/she will
+assign that ticket to a second level of support consisting of Node
+Administrators and GEs Owners. The Level 2 Help Desk team is in charge
+of providing specific high-level support for the corresponding FIWARE
+Lab Node or FIWARE Generic Enabler.
+
+In the following, we describe in more details the procedures for Level 1
+support.
+
+The Level 1 Help Desk team is the initial contact point for all incoming
+tickets, it provides support for general issues that can be easily
+solved by pointing to the FAQ, Stack Overflow, groups or other
+documentation. Moreover, its responsibility is to filter, categorise and
+forward to Level 2 support (Node Owners or GE Owners) all those tickets
+that it is not able to answer. It is also responsible for managing
+Community Account requests.
+
+The Level 1 Help Desk team is organized in 8x5 (Mon to Fri) scheduled
+shifts from 08:00 to 17:00. The team is composed by people from
+Engineering, FBK, FIWARE Foundation, Martel that are on duty on a
+respective week according to the calendar available at:
+[*https://docs.google.com/spreadsheets/d/1X9iLY9Znd3Rh-GM4qffGEHRNKqKcHZqdm-dP0rq8OVo/edit\#gid=1383004526*](https://docs.google.com/spreadsheets/d/1X9iLY9Znd3Rh-GM4qffGEHRNKqKcHZqdm-dP0rq8OVo/edit#gid=1383004526).
+Regarding INFOTEC and ITSM they are in charge of the management of the
+FIWARE Lab Mexico node, providing support to the corresponding Level 1
+Help Desk team.
+
+At the end of each day, there should be no tickets remaining in status
+“unassigned” within the FIWARE Lab Help Desk queue:
+[*http://backlog.fiware.org/lab/helpdesk*](http://backlog.fiware.org/lab/helpdesk).
+
+#### Level1 Support Requests Management
+
+The activity that each member of the L1 Support Team should be assigned
+the corresponding tickets to the proper person, assigning the
+corresponding Component and for statistical reasons assign the
+corresponding HD-Chapter, HD-Enabler and HD-Node. They are Jira issues
+attributes that need to be assigned manually when a new ticket is
+received. Usually, it is made by the FIWARE Scrum Master together with
+the process to assign the ticket to the corresponding owner in order to
+be resolved.
+
+![HD-Chapter, HD-Enabler and HD-Node attributes in issues.](image22.png)
```

Comment:
`![HD-Enabler and HD-Node attributes in issues](image22.png)`

Image to be updated in a future PR...